src/System.Private.CoreLib/PinvokeAnalyzerExceptionList.analyzerdata

```diff
@@ -1,3 +1,4 @@
 <!-- Not a bug, these APIs are in the allowed list but under kernel32 and we need it under normaliz.dll -->
 normaliz.dll!IsNormalizedString
 normaliz.dll!NormalizeString
+combase.dll!RoGetActivationFactory
```

Comment:
The right fix is to change RoGetActivationFactory to be imported from `api-ms-win-core-winrt-l1-1-0.dll`. Why not do that instead?

Reply:
@luqunl can answer... I was just porting his commit from the auto-PR, which had been sitting for days without being able to be updated.